src/test/java/ParametersTextTest.java

```diff
@@ -0,0 +1,60 @@
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+/**
+ * Created by LiudaBalatskaya on 5/10/2017.
+ */
+public class ParametersTextTest {
+    public static String FILE_PATH = System.getProperty("user.dir") + "\\src\\main\\resources\\param.txt";
+    public int result;
+    public Mathematics calculator;
+
+
+    @DataProvider(name = "provideData")
+    public Object[][] provideData() {
+        int i = 0;
+        int one;
+        int two;
+        Object[][] provideData;
+        provideData = new  Integer [7][2];
```

Comment:
Whats happened if I will add 1 line to file? Do not use hardcoded array initialization, it should be dynamic